server/src/main/java/org/apache/druid/server/security/PreResponseAuthorizationCheckFilter.java

```diff
@@ -90,7 +89,7 @@ public void doFilter(ServletRequest servletRequest, ServletResponse servletRespo
       // (e.g. OverlordServletProxy), so this is not implemented for now.
       handleAuthorizationCheckError(
           StringUtils.format(
-              "Request did not have an authorization check performed, original response status[%s].",
+              "Request did not have an authorization check performed, original response status: %s",
```

Comment:
Please do not remove interpolations from the `[]`.  It's an established standard convention for generating log messages and exception messages.